app/scripts/services/events.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('EventsService', [
+    function() {
+
+      // TODO: extract Store as a general service that can be used elsewhere.
+      // TODO: along with the auto prefix of `openshift/`, it should support
+      // many unique keys so we dont need to new Store() for each key
+      // Automaticaly namespaces all of our storage, whether in session or local
+      var namespace = 'openshift/';
```

Comment:
Nit: probably better to call this `prefix` to avoid confusion with a Kubernetes namespace

(Follow on)